src/electron-forge-import.js

```diff
@@ -0,0 +1,175 @@
+  }
+
+  await writeChanges();
+
+  if (electronName) {
+    const pruneSpinner = ora.ora('Pruning deleted modules').start();
+    await new Promise((resolve) => {
+      d('attempting to prune node_modules in:', dir);
+      const child = yarnOrNPMSpawn(hasYarn() ? [] : ['prune'], {
+        cwd: dir,
+        stdio: 'ignore',
+      });
+      child.on('exit', () => resolve());
+    });
+    pruneSpinner.succeed();
+
+    const installSpinner = ora.ora('Installing dependencies').start();
+
+    await pify(rimraf)(path.resolve(dir, 'node_modules/.bin/electron'));
+    await pify(rimraf)(path.resolve(dir, 'node_modules/.bin/electron.cmd'));
+    await pify(rimraf)(path.resolve(dir, 'node_modules', electronName));
+
+    d('installing dependencies');
+    await installDepList(dir, deps);
+    d('installing devDependencies');
+    await installDepList(dir, devDeps, true);
+    d('installing electron-prebuilt-compile');
+    await installDepList(dir, [`electron-prebuilt-compile@${electronVersion}`], false, true);
+    installSpinner.succeed();
+  }
+
+  packageJSON = JSON.parse(await fs.readFile(packageJSONPath, 'utf8'));
+
+  packageJSON.config = packageJSON.config || {};
+  packageJSON.config.forge = JSON.parse(await fs.readFile(path.resolve(__dirname, '../tmpl/package.json'))).config.forge;
+
+  await writeChanges();
+
+  const gitignoreSpinner = ora.ora('Fixing .gitignore').start();
+  if (await fs.exists(path.resolve(dir, '.gitignore'))) {
+    const gitignore = await fs.readFile(path.resolve(dir, '.gitignore'));
+    if (!gitignore.includes('out')) {
+      await fs.writeFile(path.resolve(dir, '.gitignore'), `${gitignore}\nout/`);
+    }
+  }
+  gitignoreSpinner.succeed();
+
+  if (packageJSON.babel) {
+    const babelSpinner = ora.ora('Porting original babel config').start();
+
+    let compileConfig = {};
+    const compilePath = path.resolve(dir, '.compilerc');
+    if (await fs.exists(compilePath)) {
+      compileConfig = JSON.parse(await fs.readFile(compilePath, 'utf8'));
+    }
+
+    await fs.writeFile(compilePath, JSON.stringify(Object.assign(compileConfig, {
+      'application/javascript': packageJSON.babel,
+    }, null, 2)));
+
+    babelSpinner.succeed();
+    console.info('NOTE: You might be able to remove your `.compilerc` file completely if you are only using the `es2015` and `react` presets'.yellow);
+  }
+  /* eslint-disable prefer-template */
+  console.info('\n\nWe have ATTEMPTED to convert your app to be in a format that electron-forge understands.\nNothing much will have changed but we added the' +
+  ' "electron-prebuilt-compile" '.cyan +
+  'dependency.  This is the dependency you must version bump to get newer versions of Electron.\n\n' +
+  'We also tried to import any build tooling you already had but we can\'t get everything.  You might need to convert any CLI/gulp/grunt tasks yourself.\n\n' +
+  'Thanks for using ' + 'electron-forge'.green + '!!!');
```

Comment:
I feel like this should be a multi-line templated string (if that works). Not a big fan of the `+`s and backslashes.